EKrumynas/Controllers/BlogController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using EKrumynas.Models;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using EKrumynas.Services;
+using Microsoft.AspNetCore.Routing;
+
+namespace EKrumynas.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class BlogController : ControllerBase
+    {
+        private readonly IBlogService _blogService;
+
+        public BlogController(IBlogService blogService)
+        {
+            _blogService = blogService;
+        }
+
+        [HttpGet]
+        public IList<BlogPost> GetAll()
+        {
+            var blogs = _blogService.GetAll();
+
+            return blogs ?? new List<BlogPost>();
```

Comment:
Is this necessary? Will Blog Service return null or it will only return an empty list and '... ?? new List<BlogPost>();' part is redundant?

Reply:
Same in other places.

Reply:
I don't want the controller to depend on service so I add this check. Without it if we modified the service or it unexpectedly returns null the front would break, whereas now we don't have to worry about that.